### ISSUE TITLE
Change label in i18n file of the demo shell are not reflected in the page

### ISSUE THREAD

**Issue by magemello** [Alfresco/alfresco-ng2-components#1122]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [x] Bug  
- [ ] Support request
```
**Current behavior:**
Try to change one string inside the i18n file, it doesn't shows in the page

**Expected behavior:**
Labels in the i18n folder of the demo shell should override  the original string in the components

**Comment by DenisVuyka** [comment]
see #1076

**Comment by mauriziovitale** [comment]
Should be fixed with #1076
